### PR TITLE
Move elliott 'list' commands into cli 

### PR DESCRIPTION
**Pull request by shiywang**
Move elliott 'list' commands into cli folder

```
[dev@492a2d100413 elliott]$ ./elliott -g openshift-4.2 list

2019-Oct-29 QE              OpenShift Container Platform 4.2.1 OLM Operators metadata update                 https://errata.devel.redhat.com/advisory/47337
2019-Oct-29 QE              OpenShift Container Platform 4.2.1 extras update                                 https://errata.devel.redhat.com/advisory/47336
2019-Oct-29 QE              OpenShift Container Platform 4.2.1 images update                                 https://errata.devel.redhat.com/advisory/47335
2019-Oct-29 QE              OpenShift Container Platform 4.2.1 bug fix update                                https://errata.devel.redhat.com/advisory/47334
2019-Oct-16 NEW_FILES       OpenShift Container Platform 4.2.z OLM Operators metadata update                 https://errata.devel.redhat.com/advisory/47319
2019-Oct-30 QE              OpenShift Container Platform 4.1.21 OLM Operators metadata update                https://errata.devel.redhat.com/advisory/47253
```

### CODE REVIEW

File: elliott [hunk 1]
```diff
@@ -951,16 +911,21 @@ unsigned builds.
     except ErrataException as ex:
         raise ElliottFatalError(getattr(ex, 'message', repr(ex)))
 
-## Register additional commands / groups
+
+# Register additional command groups
 cli.add_command(tarball_sources_cli)
+cli.add_command(create_placeholder)
```
Comment (vfreex) [inline elliott]: You can remove this line

Comment (shiywang) [reply elliott]: done

**Pull request merged**